src/utils/url.js

```diff
@@ -172,3 +172,9 @@ export const autocompleteUrl = (value: string = '', protocol: string, append: st
     : '';
 
 export const isValidUrl = (url: string): boolean => urlRegex({ exact: true }).test(url);
+
+export const isUrlFilePath = (url: string) =>
```

Comment:
This is certainly not robust enough.
For example, `mysite.com/welcome.php` does match it but should not be downloaded.

Usually, the distinction between a downloadable file and viewable file is made based on the mime type, and often the server can 'enforce' (or maybe more correctly 'suggest') the downloadability of a file.

Reply:
Yup! I agree it's not robust.

But to get ` mime type` (from header) we need to do a fetch (send a request), and let's say if file is 1mb, then first 1mb (and time) is gone for checking ` mime type` and if it is downloadable file then download it (again 1mb + time).